--- src/test/resources/org/checkstyle/autofix/recipe/redundantimport/defaultpackage/InputDefaultPackage.java
+++ src/test/resources/org/checkstyle/autofix/recipe/redundantimport/defaultpackage/OutputDefaultPackage.java
@@ -8,7 +8,6 @@
 package org.checkstyle.autofix.recipe.redundantimport.defaultpackage;
 
 import java.util.List;
-import java.util.List; // violation 'Duplicate import'
 
-class InputDefaultPackage {
+class OutputDefaultPackage {
 }
