--- src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/error2/InputError2.java
+++ src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/error2/OutputError2.java
@@ -12,20 +12,20 @@
  * The constructor InputError2(String b) ends up at the violation location after first cycle.
  * It should remain at the same position after second cycle.
  */
-public class InputError2 {
+public class OutputError2 {
 
-    InputError2() {}
+    OutputError2() {}
+
+    OutputError2(int a) {}
+
+    OutputError2(String b) {}
+
+    OutputError2(int a, String b) {}
 
     int a;
 
-    InputError2(int a) {} // violation 'Constructors should be grouped together'
-
     String b;
 
-    InputError2(String b) {} // violation 'Constructors should be grouped together'
-
     void foo() {}
 
-    InputError2(int a, String b) {} // violation 'Constructors should be grouped together'
-
 }
