--- src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/multiplefiles1/InputMultipleFiles1.java
+++ src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/multiplefiles1/OutputMultipleFiles1.java
@@ -17,16 +17,15 @@
  * Multiple files.
  * Verifies that violating constructors from the correct files are moved.
  */
-public class InputMultipleFiles1 {
+public class OutputMultipleFiles1 {
 
     int x;
 
-    InputMultipleFiles1() {}
+    OutputMultipleFiles1() {}
 
-    InputMultipleFiles1(String s) {}
+    OutputMultipleFiles1(String s) {}
+
+    OutputMultipleFiles1(int x) {}
 
     void foo() {}
-
-    InputMultipleFiles1(int x) {}
-    // violation above 'Constructors should be grouped together.*'
 }
