--- src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/ordermultipleclasses/InputOrderMultipleClasses.java
+++ src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/ordermultipleclasses/OutputOrderMultipleClasses.java
@@ -11,24 +11,20 @@
 
 package org.checkstyle.autofix.recipe.constructorsdeclarationgrouping.ordermultipleclasses;
 
-public class InputOrderMultipleClasses {
+public class OutputOrderMultipleClasses {
 
-    InputOrderMultipleClasses(String s) {}
+    OutputOrderMultipleClasses() {}
 
-    InputOrderMultipleClasses() {}
-    // violation above 'Constructors should be ordered.*'
+    OutputOrderMultipleClasses(String s) {}
 
-    InputOrderMultipleClasses(int x) {}
-    // violation above 'Constructors should be ordered.*'
+    OutputOrderMultipleClasses(int x) {}
 
     static class Inner {
 
-        Inner(String s, int x) {}
+        Inner() {}
 
         Inner(String s) {}
-        // violation above 'Constructors should be ordered.*'
 
-        Inner() {}
-        // violation above 'Constructors should be ordered.*'
+        Inner(String s, int x) {}
     }
 }
