--- src/test/resources/org/checkstyle/autofix/recipe/unnecessaryparentheses/multifile2/InputMultiFile2.java
+++ src/test/resources/org/checkstyle/autofix/recipe/unnecessaryparentheses/multifile2/OutputMultiFile2.java
@@ -8,7 +8,7 @@
 
 package org.checkstyle.autofix.recipe.unnecessaryparentheses.multifile;
 
-public class InputMultiFile2 {
+public class OutputMultiFile2 {
     public int square2(int a, int b, int c) {
         return (a + b) * c;
     }
