--- src/test/resources/org/checkstyle/autofix/recipe/missingswitchdefault/simple/InputSimple.java
+++ src/test/resources/org/checkstyle/autofix/recipe/missingswitchdefault/simple/OutputSimple.java
@@ -7,11 +7,13 @@
 */
 package org.checkstyle.autofix.recipe.missingswitchdefault.simple;
 
-public class InputSimple {
+public class OutputSimple {
     void foo(int x) {
-        switch (x) { // violation 'switch without "default" clause.'
+        switch (x) {
             case 1:
                 break;
+            default:
+                break;
         }
     }
 }
