--- src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/noconstructors/InputNoConstructors.java
+++ src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/noconstructors/OutputNoConstructors.java
@@ -10,7 +10,7 @@
 package org.checkstyle.autofix.recipe.constructorsdeclarationgrouping.noconstructors;
 
 /** There are no constructors at all. Verifies that no modification is made. */
-public class InputNoConstructors {
+public class OutputNoConstructors {
 
     private void doNothing() {}
 
