--- src/test/resources/org/checkstyle/autofix/recipe/requirethis/anonymousinanonymous/InputAnonymousInAnonymous.java
+++ src/test/resources/org/checkstyle/autofix/recipe/requirethis/anonymousinanonymous/OutputAnonymousInAnonymous.java
@@ -10,7 +10,7 @@
 
 package org.checkstyle.autofix.recipe.requirethis.anonymousinanonymous;
 
-public class InputAnonymousInAnonymous {
+public class OutputAnonymousInAnonymous {
     class Wrapper {
         private int buffer;
         
@@ -21,7 +21,7 @@
                     Runnable r2 = new Runnable() {
                         @Override
                         public void run() {
-                            buffer = 5; // violation 'Reference to instance variable 'buffer' needs "Wrapper.this."'
+                            Wrapper.this.buffer = 5;
                         }
                     };
                 }
