diff --git src/test/resources/org/checkstyle/autofix/recipe/header/headercomments/InputHeaderComments.java src/test/resources/org/checkstyle/autofix/recipe/header/headercomments/OutputHeaderComments.java
index fa0cfcb..3894f18 100644
--- src/test/resources/org/checkstyle/autofix/recipe/header/headercomments/InputHeaderComments.java
+++ src/test/resources/org/checkstyle/autofix/recipe/header/headercomments/OutputHeaderComments.java
@@ -1,3 +1,10 @@
+///////////////////////////////////////////////////////////////////////////////////////////////
+// Unit test for checkstyle-openrewrite-recipes.
+// Dated: XX.XX.XX
+// Copyright (C) 2025 Authors. Licensed under Apache 2.0.
+// This file is part of the Checkstyle OpenRewrite test suite.
+///////////////////////////////////////////////////////////////////////////////////////////////
+
 ///////////////////////////////////////////////////////////////////////////////////////////////
 // Unit test for checkstyle-openrewrite-recipes.
 // Dated: 11.07.25
@@ -6,5 +13,5 @@
 
 package org.checkstyle.autofix.recipe.header.headercomments;
 
-public class InputHeaderComments {
+public class OutputHeaderComments {
 }
