--- src/test/resources/org/checkstyle/autofix/recipe/finallocalvariable/localvariablecheckswitchassignment/InputLocalVariableCheckSwitchAssignment.java
+++ src/test/resources/org/checkstyle/autofix/recipe/finallocalvariable/localvariablecheckswitchassignment/OutputLocalVariableCheckSwitchAssignment.java
@@ -9,7 +9,7 @@
 */
 package org.checkstyle.autofix.recipe.finallocalvariable.localvariablecheckswitchassignment;
 
-public class InputLocalVariableCheckSwitchAssignment {
+public class OutputLocalVariableCheckSwitchAssignment {
     private static final int staticValue = 2;
     private static final int staticField = switch(staticValue) {
         case 0 -> -1;
@@ -18,7 +18,7 @@
     };
 
     public void InputFinalLocalVariableCheckSwitchAssignment() throws Exception {
-        int a = 0; // violation, "Variable 'a' should be declared final"
+        final int a = 0;
         final int b = 0;
         int c = 10;
         int d = 0;
@@ -41,9 +41,9 @@
     public void foo() throws Exception {
         final int a = 0;
 
-        int b = switch(a) { // violation, "Variable 'b' should be declared final"
+        final int b = switch(a) {
             case 0 -> {
-                int x = 5; // violation, "Variable 'x' should be declared final"
+                final int x = 5;
                 int y = 6;
                 if (a == 2) {
                     y = 7;
@@ -69,7 +69,7 @@
     }
 
     String statement1(int t) {
-        String res; // violation, "Variable 'res' should be declared final"
+        final String res;
 
         switch (t) {
             case 1 -> {
@@ -89,7 +89,7 @@
     }
 
     void switch_rules(MyEnum value) {
-        String res; // violation, "Variable 'res' should be declared final"
+        final String res;
         switch (value) {
             case a -> throw new RuntimeException();
             case b -> res = "2";
