--- src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/withprefixwhitespaces/InputWithPrefixWhitespaces.java
+++ src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/withprefixwhitespaces/OutputWithPrefixWhitespaces.java
@@ -13,19 +13,17 @@
  * Contains ungrouped constructors with irregular leading whitespaces.
  * Verifies that leading whitespaces are carried along.
  */
-public class InputWithPrefixWhitespaces {
+public class OutputWithPrefixWhitespaces {
 
     int x;
-InputWithPrefixWhitespaces() {}
+OutputWithPrefixWhitespaces() {}
 
-    InputWithPrefixWhitespaces(String s) {}
+    OutputWithPrefixWhitespaces(String s) {}
+
+        OutputWithPrefixWhitespaces(int x) {}
+
+
+    OutputWithPrefixWhitespaces(String s, int x) {}
 
             void foo() {}
-
-        InputWithPrefixWhitespaces(int x) {}
-    // violation above 'Constructors should be grouped together.*'
-
-
-    InputWithPrefixWhitespaces(String s, int x) {}
-    // violation above 'Constructors should be grouped together.*'
 }
