--- src/test/resources/org/checkstyle/autofix/recipe/avoidstaticimport/starstaticimport/InputStarStaticImport.java
+++ src/test/resources/org/checkstyle/autofix/recipe/avoidstaticimport/starstaticimport/OutputStarStaticImport.java
@@ -6,12 +6,11 @@
 </module>
 */
 package org.checkstyle.autofix.recipe.avoidstaticimport.starstaticimport;
-
-import static java.util.Collections.*; // violation 'Using a static member import should be avoided - java.util.Collections.*.'
+import java.util.Collections;
 import java.util.List;
 
-public class InputStarStaticImport {
+public class OutputStarStaticImport {
     void process(List<String> list) {
-        sort(list);
+        Collections.sort(list);
     }
 }
