--- src/test/resources/org/checkstyle/autofix/recipe/finalclass/publicconstructor/InputPublicConstructor.java
+++ src/test/resources/org/checkstyle/autofix/recipe/finalclass/publicconstructor/OutputPublicConstructor.java
@@ -8,7 +8,7 @@
 
 package org.checkstyle.autofix.recipe.finalclass.publicconstructor;
 
-public class InputPublicConstructor {
-    public InputPublicConstructor() {
+public class OutputPublicConstructor {
+    public OutputPublicConstructor() {
     }
 }
