--- src/test/resources/org/checkstyle/autofix/recipe/hexliterialcase/hexliteralcasetwo/InputHexLiteralCaseTwo.java
+++ src/test/resources/org/checkstyle/autofix/recipe/hexliterialcase/hexliteralcasetwo/OutputHexLiteralCaseTwo.java
@@ -9,12 +9,12 @@
 
 package org.checkstyle.autofix.recipe.hexliterialcase.hexliteralcasetwo;
 
-public class InputHexLiteralCaseTwo {
-    float f = 0x1.fp1f;  // violation  'Should use uppercase hexadecimal letters.'
-    double d = 0x1abcde.fp1d; // violation  'Should use uppercase hexadecimal letters.'
+public class OutputHexLiteralCaseTwo {
+    float f = 0x1.Fp1f;
+    double d = 0x1ABCDE.Fp1d;
 
-    float f1 = 0x1a_e_f.1p1f; // violation  'Should use uppercase hexadecimal letters.'
-    double d1 = 0x1a_b_c_d_e.1p1d; // violation  'Should use uppercase hexadecimal letters.'
+    float f1 = 0x1A_E_F.1p1f;
+    double d1 = 0x1A_B_C_D_E.1p1d;
 
-    float f2 = 0xab.1P1f; // violation  'Should use uppercase hexadecimal letters.'
+    float f2 = 0xAB.1P1f;
 }
