--- src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/multiplefiles2/InputMultipleFiles2.java
+++ src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/multiplefiles2/OutputMultipleFiles2.java
@@ -17,15 +17,15 @@
  * Multiple files.
  * Verifies that violating constructors from the correct files are moved.
  */
-public class InputMultipleFiles2 {
+public class OutputMultipleFiles2 {
 
     int x;
 
-    InputMultipleFiles2() {}
+    OutputMultipleFiles2() {}
 
-    InputMultipleFiles2(String s) {}
+    OutputMultipleFiles2(String s) {}
 
     void foo() {}
 
-    InputMultipleFiles2(int x) {}
+    OutputMultipleFiles2(int x) {}
 }
