--- src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/multiplefiles2/InputMultipleFiles2.java
+++ src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/multiplefiles2/OutputMultipleFiles2.java
@@ -13,15 +13,15 @@
 
 package org.checkstyle.autofix.recipe.constructorsdeclarationgrouping.multiplefiles2;
 
-public class InputMultipleFiles2 {
+public class OutputMultipleFiles2 {
 
     int x;
 
-    InputMultipleFiles2() {}
+    OutputMultipleFiles2() {}
 
-    InputMultipleFiles2(String s) {}
+    OutputMultipleFiles2(String s) {}
 
     void foo() {}
 
-    InputMultipleFiles2(int x) {}
+    OutputMultipleFiles2(int x) {}
 }
