diff --git src/test/resources/org/checkstyle/autofix/recipe/upperell/hexoctalliteral/InputHexOctalLiteral.java src/test/resources/org/checkstyle/autofix/recipe/upperell/hexoctalliteral/OutputHexOctalLiteral.java
index bc94922..52ed9a2 100644
--- src/test/resources/org/checkstyle/autofix/recipe/upperell/hexoctalliteral/InputHexOctalLiteral.java
+++ src/test/resources/org/checkstyle/autofix/recipe/upperell/hexoctalliteral/OutputHexOctalLiteral.java
@@ -13,19 +13,17 @@
 
 package org.checkstyle.autofix.recipe.upperell.hexoctalliteral;
 
-public class InputHexOctalLiteral {
-    private long hexLower = 0x1ABCl;    // violation 'Should use uppercase 'L'.'
-    private long hexUpper = 0X2DEFl;    // violation 'Should use uppercase 'L'.'
-    private long octal = 0777l;         // violation 'Should use uppercase 'L'.'
-    private long binary = 0b1010l;      // violation 'Should use uppercase 'L'.'
+public class OutputHexOctalLiteral {
+    private long hexLower = 0x1ABCL;
+    private long hexUpper = 0X2DEFL;
+    private long octal = 0777L;
+    private long binary = 0b1010L;
     private long decimal = // comment preceding a target element
-                           12345l;      // violation 'Should use uppercase 'L'.'
+                           12345L;
 
     public void calculateValues() {
-        // violation below, 'Should use uppercase 'L'.'
-        long hexResult = 0xDEADBEEFl + 0xDEADBEFl; //suppressed violation for 0xDEADBEFl
-        // violation below, 'Should use uppercase 'L'.'
-        long octalResult = 01234l + 0xDEADBEEFl;   //suppressed violation for 01234l
-        long binaryResult = 0b11110000l;           // violation 'Should use uppercase 'L'.'
+        long hexResult = 0xDEADBEEFL + 0xDEADBEFl; //suppressed violation for 0xDEADBEFl
+        long octalResult = 01234l + 0xDEADBEEFL;   //suppressed violation for 01234l
+        long binaryResult = 0b11110000L;
     }
 }
