--- src/test/resources/org/checkstyle/autofix/recipe/finallocalvariable/localvariablecheckswitchexpressions/InputLocalVariableCheckSwitchExpressions.java
+++ src/test/resources/org/checkstyle/autofix/recipe/finallocalvariable/localvariablecheckswitchexpressions/OutputLocalVariableCheckSwitchExpressions.java
@@ -9,10 +9,10 @@
 */
 package org.checkstyle.autofix.recipe.finallocalvariable.localvariablecheckswitchexpressions;
 
-public class InputLocalVariableCheckSwitchExpressions {
+public class OutputLocalVariableCheckSwitchExpressions {
     void foo1() throws Exception {
 
-        Exception e; // violation, "Variable 'e' should be declared final"
+        final Exception e;
 
         final int a = (int) Math.random();
         final int b = (int) Math.random();
@@ -50,7 +50,7 @@
 
     void foo2() throws Exception {
 
-        Exception e; // violation, "Variable 'e' should be declared final"
+        final Exception e;
 
         final int a = (int) Math.random();
         final int b = (int) Math.random();
@@ -88,7 +88,7 @@
 
     void foo3() throws Exception {
 
-        Exception e; // violation, "Variable 'e' should be declared final"
+        final Exception e;
 
         final int a = (int) Math.random();
         final int b = (int) Math.random();
@@ -122,7 +122,7 @@
 
     void foo4() throws Exception {
 
-        Exception e; // violation, "Variable 'e' should be declared final"
+        final Exception e;
 
         final int a = (int) Math.random();
         final int b = (int) Math.random();
