--- src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/suppressedviolations/InputSuppressedViolations.java
+++ src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/suppressedviolations/OutputSuppressedViolations.java
@@ -17,18 +17,17 @@
  * Not all constructors are grouped together, but violations are suppressed.
  * Verifies that no modification is made.
  */
-public class InputSuppressedViolations {
+public class OutputSuppressedViolations {
 
     int x;
 
-    InputSuppressedViolations() {}
+    OutputSuppressedViolations() {}
 
-    InputSuppressedViolations(String s) {}
+    OutputSuppressedViolations(String s) {}
+
+    OutputSuppressedViolations(String s, int x) {}
 
     void foo() {}
 
-    InputSuppressedViolations(String s, int x) {}
-    // violation above 'Constructors should be grouped together.*'
-
-    InputSuppressedViolations(int x) {}
+    OutputSuppressedViolations(int x) {}
 }
