--- src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/ordernoviolation/InputOrderNoViolation.java
+++ src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/ordernoviolation/OutputOrderNoViolation.java
@@ -11,11 +11,11 @@
 
 package org.checkstyle.autofix.recipe.constructorsdeclarationgrouping.ordernoviolation;
 
-public class InputOrderNoViolation {
+public class OutputOrderNoViolation {
 
-    InputOrderNoViolation() {}
+    OutputOrderNoViolation() {}
 
-    InputOrderNoViolation(int x) {}
+    OutputOrderNoViolation(int x) {}
 
-    InputOrderNoViolation(String s) {}
+    OutputOrderNoViolation(String s) {}
 }
