--- src/test/resources/org/checkstyle/autofix/recipe/avoidstarimport/inheritedinstancefieldchild/InputInheritedInstanceFieldChild.java
+++ src/test/resources/org/checkstyle/autofix/recipe/avoidstarimport/inheritedinstancefieldchild/OutputInheritedInstanceFieldChild.java
@@ -9,10 +9,12 @@
 
 package org.checkstyle.autofix.recipe.avoidstarimport.inheritedinstancefieldchild;
 
-import java.util.*; // violation 'Using the .* form of import should be avoided'
-import org.checkstyle.autofix.recipe.avoidstarimport.inheritedinstancefieldparent.*; // violation 'Using the .* form of import should be avoided'
+import org.checkstyle.autofix.recipe.avoidstarimport.inheritedinstancefieldparent.InputInheritedInstanceFieldParent;
 
-public class InputInheritedInstanceFieldChild extends InputInheritedInstanceFieldParent {
+import java.util.ArrayList;
+import java.util.List;
+
+public class OutputInheritedInstanceFieldChild extends InputInheritedInstanceFieldParent {
     private final List<String> values = new ArrayList<>();
 
     int size() {
