--- src/test/resources/org/checkstyle/autofix/recipe/emptystatement/emptywhileloop/InputEmptyWhileLoop.java
+++ src/test/resources/org/checkstyle/autofix/recipe/emptystatement/emptywhileloop/OutputEmptyWhileLoop.java
@@ -9,12 +9,12 @@
 
 package org.checkstyle.autofix.recipe.emptystatement.emptywhileloop;
 
-public class InputEmptyWhileLoop {
+public class OutputEmptyWhileLoop {
     void method() {
-        while (true); // violation 'Empty statement.'
+        while (true) {}
     }
 
     void conditionLoop(boolean condition) {
-        while (condition); // violation 'Empty statement.'
+        while (condition) {}
     }
 }
