--- src/test/resources/org/checkstyle/autofix/recipe/useenhancedswitch/emptycaseanddefault/InputEmptyCaseAndDefault.java
+++ src/test/resources/org/checkstyle/autofix/recipe/useenhancedswitch/emptycaseanddefault/OutputEmptyCaseAndDefault.java
@@ -8,16 +8,13 @@
 
 package org.checkstyle.autofix.recipe.useenhancedswitch.emptycaseanddefault;
 
-public class InputEmptyCaseAndDefault {
+public class OutputEmptyCaseAndDefault {
     private boolean isGroupReady(int status) {
-        // violation below, 'Switch can be replaced with enhanced switch'
         switch (status) {
-            case 1:
-            case 2:
-            case 3:
-            case 4:
+            case 1, 2, 3, 4 -> {
                 return false;
-            default:
+            }
+            default -> {}
                 // Empty
         }
         return true;
