--- src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/noviolation/InputNoViolation.java
+++ src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/noviolation/OutputNoViolation.java
@@ -12,15 +12,15 @@
  * All constructors are already grouped together with no violations.
  * Verifies that the Recipe makes no changes when the code is already compliant.
  */
-public class InputNoViolation {
+public class OutputNoViolation {
 
     int x;
 
-    InputNoViolation() {}
+    OutputNoViolation() {}
 
-    InputNoViolation(String s) {}
+    OutputNoViolation(String s) {}
 
-    InputNoViolation(int x) {}
+    OutputNoViolation(int x) {}
 
     void foo() {}
 
