--- src/test/resources/org/checkstyle/autofix/recipe/avoidstarimport/multiplerequiredstarimports/InputMultipleRequiredStarImports.java
+++ src/test/resources/org/checkstyle/autofix/recipe/avoidstarimport/multiplerequiredstarimports/OutputMultipleRequiredStarImports.java
@@ -9,14 +9,17 @@
 
 package org.checkstyle.autofix.recipe.avoidstarimport.multiplerequiredstarimports;
 
-import java.time.*; // violation 'Using the .* form of import should be avoided'
-import java.util.*; // violation 'Using the .* form of import should be avoided'
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+import java.util.Map;
 
-import static java.lang.Integer.*; // violation 'Using the .* form of import should be avoided'
-import static java.time.Instant.*; // violation 'Using the .* form of import should be avoided'
-import static java.util.Collections.*; // violation 'Using the .* form of import should be avoided'
+import static java.lang.Integer.MAX_VALUE;
+import static java.time.Instant.EPOCH;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonMap;
 
-public class InputMultipleRequiredStarImports {
+public class OutputMultipleRequiredStarImports {
     private final List<Duration> durations = emptyList();
     private final Map<String, Instant> values = singletonMap("epoch", EPOCH);
     private final int maxValue = MAX_VALUE;
