--- src/test/resources/org/checkstyle/autofix/recipe/annotationonsameline/basicclassannotation/InputBasicClassAnnotation.java
+++ src/test/resources/org/checkstyle/autofix/recipe/annotationonsameline/basicclassannotation/OutputBasicClassAnnotation.java
@@ -8,6 +8,5 @@
 
 package org.checkstyle.autofix.recipe.annotationonsameline.basicclassannotation;
 
-@Deprecated // violation, "should be on the same line with its target."
-public class InputBasicClassAnnotation {
+@Deprecated public class OutputBasicClassAnnotation {
 }
