--- src/test/resources/org/checkstyle/autofix/recipe/unusedlocalvariable/multiplesideeffects/InputMultipleSideEffects.java
+++ src/test/resources/org/checkstyle/autofix/recipe/unusedlocalvariable/multiplesideeffects/OutputMultipleSideEffects.java
@@ -8,12 +8,12 @@
 
 package org.checkstyle.autofix.recipe.unusedlocalvariable.multiplesideeffects;
 
-public class InputMultipleSideEffects {
+public class OutputMultipleSideEffects {
 
     public void method() {
         int a = 0;
         // Comment before multiple side effects
-        int unused1 = ++a, unused2 = a++; // 2 violations
+        ++a; a++;
         System.out.println(a);
     }
 }
