--- src/test/resources/org/checkstyle/autofix/recipe/finallocalvariable/localvariablethree/InputLocalVariableThree.java
+++ src/test/resources/org/checkstyle/autofix/recipe/finallocalvariable/localvariablethree/OutputLocalVariableThree.java
@@ -9,16 +9,16 @@
 */
 package org.checkstyle.autofix.recipe.finallocalvariable.localvariablethree;
 
-public class InputLocalVariableThree {
+public class OutputLocalVariableThree {
     class class2 {
         public void method1() {
-            int x; // violation, "Variable 'x' should be declared final"
+            final int x;
             x = 3;
         }
 
         public void method2() {
             for (int i = 0; i < 5; i++) {
-                int x; // violation, "Variable 'x' should be declared final"
+                final int x;
                 x = 3;
             }
             int y;
@@ -39,14 +39,14 @@
                 m = 0;
             } while (false);
             do {
-                int n; // violation, "Variable 'n' should be declared final"
+                final int n;
                 n = 0;
             } while (true);
         }
 
         private void foo() {
-            int q; // violation, "Variable 'q' should be declared final"
-            int w; // violation, "Variable 'w' should be declared final"
+            final int q;
+            final int w;
             int e;
             q = 1;
             w = 1;
@@ -55,8 +55,8 @@
             class Local {
                 void bar() {
                     int q;
-                    int w; // violation, "Variable 'w' should be declared final"
-                    int e; // violation, "Variable 'e' should be declared final"
+                    final int w;
+                    final int e;
                     q = 1;
                     q = 2;
                     w = 1;
@@ -77,7 +77,7 @@
                 i++;
             }
             while (true) {
-                int n; // violation, "Variable 'n' should be declared final"
+                final int n;
                 n = 0;
             }
         }
@@ -90,7 +90,7 @@
                 r = 0;
             }
             for (int a : array) {
-                int t; // violation, "Variable 't' should be declared final"
+                final int t;
                 t = 0;
             }
         }
@@ -100,7 +100,7 @@
         public void method(final int i) {
             switch (i) {
                 case 1:
-                    int foo = 1;    // violation, "Variable 'foo' should be declared final"
+                    final int foo = 1;
                     break;
                 default:
             }
