--- src/test/resources/org/checkstyle/autofix/recipe/redundantimport/complexcase/InputComplexCase.java
+++ src/test/resources/org/checkstyle/autofix/recipe/redundantimport/complexcase/OutputComplexCase.java
@@ -8,15 +8,11 @@
 */
 
 package org.checkstyle.autofix.recipe.redundantimport.complexcase;
-
-import org.checkstyle.autofix.recipe.redundantimport.complexcase.*; // violation 'Redundant import from the same package'
 import java.io.*;
-    import java.lang.*; // violation 'Redundant import from the java.lang package'
 
 import java.util.List;
-import java.util.List; // violation 'Duplicate import'
 import java.util.HashMap;
-import static java.lang.Math.E; import java.lang.String; // violation 'Redundant import from the java.lang package'
+import static java.lang.Math.E;
 import java.util.Iterator;
 import java.util.Enumeration;
 import java.util.Arrays;
@@ -28,7 +24,6 @@
 
 import static java.io.File.listRoots;
 import static javax.swing.WindowConstants.*;
-import static javax.swing.WindowConstants.*; // violation 'Duplicate import'
 import static java.io.File.createTempFile;
 import static java.io.File.pathSeparator;
 import static org.checkstyle.autofix.recipe.redundantimport.complexcase.InputComplexCase.myStaticMethod;
@@ -43,6 +38,6 @@
 
 import static java.lang.Math.PI;
 
-class InputComplexCase {
+class OutputComplexCase {
     public static void myStaticMethod() {}
 }
