--- src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/idempotent1/InputIdempotent1.java
+++ src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/idempotent1/OutputIdempotent1.java
@@ -18,18 +18,17 @@
  * must not be wrongly moved again in a second cycle. Tests that XPath-suppressed
  * constructors at violation positions are correctly ignored.
  */
-public class InputIdempotent1 {
+public class OutputIdempotent1 {
 
     int x;
 
-    InputIdempotent1() {}
+    OutputIdempotent1() {}
 
-    InputIdempotent1(String s) {}
+    OutputIdempotent1(String s) {}
+
+    OutputIdempotent1(String s, int x) {}
 
     void foo() {}
 
-    InputIdempotent1(int x) {}
-
-    InputIdempotent1(String s, int x) {}
-    // violation above 'Constructors should be grouped together.*'
+    OutputIdempotent1(int x) {}
 }
