--- src/test/resources/org/checkstyle/autofix/recipe/finalclass/protectedinnerclass/InputProtectedInnerClass.java
+++ src/test/resources/org/checkstyle/autofix/recipe/finalclass/protectedinnerclass/OutputProtectedInnerClass.java
@@ -8,8 +8,8 @@
 
 package org.checkstyle.autofix.recipe.finalclass.protectedinnerclass;
 
-class InputProtectedInnerClass {
-    protected class Inner { // violation 'Class Inner should be declared as final.'
+class OutputProtectedInnerClass {
+    protected final class Inner {
         private Inner() {
         }
     }
