--- src/test/resources/org/checkstyle/autofix/recipe/nowhitespaceafter/tokens/InputTokens.java
+++ src/test/resources/org/checkstyle/autofix/recipe/nowhitespaceafter/tokens/OutputTokens.java
@@ -9,39 +9,39 @@
 */
 package org.checkstyle.autofix.recipe.nowhitespaceafter.tokens;
 
-class InputTokens {
+class OutputTokens {
     public void dotOperator(String s) {
         Integer.parseInt(s);
-        Integer . parseInt(s); // violation ''.' is followed by whitespace.'
+        Integer .parseInt(s);
         hashCode();
     }
 
     public void arrayDec() {
         int[] arr = new int[2];
-        int [] array; // violation ''int' is followed by whitespace.'
-        arr [ 0 ] = 1; // violation ''arr' is followed by whitespace.'
-        int[] a = { 1, 2 }; // violation 'is followed by whitespace.'
-        int[] emptyArray = { }; // violation 'is followed by whitespace.'
+        int[] array;
+        arr[ 0 ] = 1;
+        int[] a = {1, 2 };
+        int[] emptyArray = {};
     }
 
     public void fieldAccess() {
-        System . out.println(); // violation ''.' is followed by whitespace.'
+        System .out.println();
     }
 
     public void bitwiseNot(int a) {
-        a = ~ a; // violation ''~' is followed by whitespace.'
-        a = + a; // violation 'is followed by whitespace.'
-        a = - a; // violation ''-' is followed by whitespace.'
-        boolean b = ! (a > 0); // violation ''!' is followed by whitespace.'
+        a = ~a;
+        a = +a;
+        a = -a;
+        boolean b = !(a > 0);
     }
 
     public void incDec(int a) {
-        ++ a; // violation 'is followed by whitespace.'
-        -- a; // violation ''--' is followed by whitespace.'
-        Object obj2 = (Object) a; // violation 'is followed by whitespace.'
+        ++a;
+        --a;
+        Object obj2 = (Object)a;
     }
 
-    @ SuppressWarnings("abc") // violation ''@' is followed by whitespace.'
+    @SuppressWarnings("abc")
     public void annotationSpace() {
     }
 
@@ -61,7 +61,7 @@
                 1, 2 };
         Object o2 = (Object)
                 a;
-        synchronized (this) {} // violation 'is followed by whitespace.'
+        synchronized(this) {}
         java.lang.
                 String s2;
     }
@@ -78,21 +78,21 @@
 
     public void nestedViolations() {
         final int x1 = 1;
-        @MyAnnotation(- x1) // violation ''-' is followed by whitespace.'
+        @MyAnnotation(-x1)
         int x_tmp;
         int[] arr = new int[2];
-        arr[- x1] = 1; // violation ''-' is followed by whitespace.'
-        int x_val = new java.awt.Point(- x1, 0).x; // violation ''-' is followed by whitespace.'
-        System.out.println(- x1); // violation ''-' is followed by whitespace.'
-        int[] arr2 = new int[] {- x1}; // violation ''-' is followed by whitespace.'
-        synchronized(java.lang.Integer.valueOf(- x1)) {} // violation ''-' is followed by whitespace.'
-        Object o3 = (Object)(- x1); // violation ''-' is followed by whitespace.'
-        int x2 = ~(- x1); // violation ''-' is followed by whitespace.'
-        int @MyAnnotation(- x1) [] annotatedArray; // violation ''-' is followed by whitespace.'
+        arr[-x1] = 1;
+        int x_val = new java.awt.Point(-x1, 0).x;
+        System.out.println(-x1);
+        int[] arr2 = new int[] {-x1};
+        synchronized(java.lang.Integer.valueOf(-x1)) {}
+        Object o3 = (Object)(-x1);
+        int x2 = ~(-x1);
+        int @MyAnnotation(-x1) [] annotatedArray;
     }
     public void methodReference() {
-        java.util.function.Supplier<String> s = String:: new; // violation ''::' is followed by whitespace.'
-        java.util.function.IntFunction<int[]> s4 = int []::new; // violation ''int' is followed by whitespace.'
+        java.util.function.Supplier<String> s = String::new;
+        java.util.function.IntFunction<int[]> s4 = int[]::new;
         java.util.function.Supplier<String> s3 = String::
                 new;
     }
