--- src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/orderwithgroupingviolationsuppressed/InputOrderWithGroupingViolationSuppressed.java
+++ src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/orderwithgroupingviolationsuppressed/OutputOrderWithGroupingViolationSuppressed.java
@@ -25,16 +25,15 @@
  * (which has an ordering violation) is moved to the group. The 0-param
  * constructor stays in place since it has no violations at all.
  */
-public class InputOrderWithGroupingViolationSuppressed {
+public class OutputOrderWithGroupingViolationSuppressed {
 
-    InputOrderWithGroupingViolationSuppressed(int x) {}
+    OutputOrderWithGroupingViolationSuppressed() {}
 
-    InputOrderWithGroupingViolationSuppressed(int x, boolean b) {}
+    OutputOrderWithGroupingViolationSuppressed(int x) {}
+
+    OutputOrderWithGroupingViolationSuppressed(int x, boolean b) {}
 
     void foo() {}
 
-    InputOrderWithGroupingViolationSuppressed(String s, int x, int y) {}
-
-    InputOrderWithGroupingViolationSuppressed() {}
-    // violation above 'Constructors should be ordered.*'
+    OutputOrderWithGroupingViolationSuppressed(String s, int x, int y) {}
 }
