--- src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/separatedbycomment/InputSeparatedByComment.java
+++ src/test/resources/org/checkstyle/autofix/recipe/constructorsdeclarationgrouping/separatedbycomment/OutputSeparatedByComment.java
@@ -13,14 +13,14 @@
  * According to the rule, comments between constructors are allowed.
  * Verifies that no modification is made.
  */
-public class InputSeparatedByComment {
+public class OutputSeparatedByComment {
 
-    InputSeparatedByComment() {}
+    OutputSeparatedByComment() {}
 
     // Default constructor above, parameterized below.
-    InputSeparatedByComment(String s) {}
+    OutputSeparatedByComment(String s) {}
 
     /* Another comment block. */
-    InputSeparatedByComment(int x) {}
+    OutputSeparatedByComment(int x) {}
 
 }
