--- src/test/resources/org/checkstyle/autofix/recipe/annotationonsameline/multiannotationsparse/InputMultiAnnotationSparse.java
+++ src/test/resources/org/checkstyle/autofix/recipe/annotationonsameline/multiannotationsparse/OutputMultiAnnotationSparse.java
@@ -8,11 +8,10 @@
 
 package org.checkstyle.autofix.recipe.annotationonsameline.multiannotationsparse;
 
-public class InputMultiAnnotationSparse {
+public class OutputMultiAnnotationSparse {
     @interface Ann1 {}
     @interface Ann2 {}
 
-    @Ann1 @Ann2 // 2 violations
-    class Target {
+    @Ann1 @Ann2 class Target {
     }
 }
