--- src/test/resources/org/checkstyle/autofix/recipe/annotationlocation/complexannotationlocation/InputComplexAnnotationLocation.java
+++ src/test/resources/org/checkstyle/autofix/recipe/annotationlocation/complexannotationlocation/OutputComplexAnnotationLocation.java
@@ -10,23 +10,30 @@
 
 package org.checkstyle.autofix.recipe.annotationlocation.complexannotationlocation;
 
-@Deprecated @SuppressWarnings("unchecked") // violation "Annotation 'Deprecated' should be alone on line."
-// violation above "Annotation 'SuppressWarnings' should be alone on line."
-class InputComplexAnnotationLocation {
+@Deprecated
+@SuppressWarnings("unchecked")
+class OutputComplexAnnotationLocation {
 
-    @Deprecated public void testModifiers() {} // violation "Annotation 'Deprecated' should be alone on line."
+    @Deprecated
+    public void testModifiers() {}
 
-    @Deprecated InputComplexAnnotationLocation() {} // violation "Annotation 'Deprecated' should be alone on line."
+    @Deprecated
+    OutputComplexAnnotationLocation() {}
 
-    @Deprecated public int testVarModifiers; // violation "Annotation 'Deprecated' should be alone on line."
+    @Deprecated
+    public int testVarModifiers;
 
-    @Deprecated void testNoModifiers() {} // violation "Annotation 'Deprecated' should be alone on line."
+    @Deprecated
+    void testNoModifiers() {}
 
-    @Deprecated int testNoVarModifiers; // violation "Annotation 'Deprecated' should be alone on line."
+    @Deprecated
+    int testNoVarModifiers;
 
-    @Deprecated public static void multiModMethod() {} // violation "Annotation 'Deprecated' should be alone on line."
+    @Deprecated
+    public static void multiModMethod() {}
 
-    @Deprecated public static int multiModVar; // violation "Annotation 'Deprecated' should be alone on line."
+    @Deprecated
+    public static int multiModVar;
 
     @Deprecated
     public    void   alreadyCorrect() {}
