--- src/test/resources/org/checkstyle/autofix/recipe/emptyforiteratorpad/optionspace/InputOptionSpace.java
+++ src/test/resources/org/checkstyle/autofix/recipe/emptyforiteratorpad/optionspace/OutputOptionSpace.java
@@ -10,9 +10,9 @@
 
 package org.checkstyle.autofix.recipe.emptyforiteratorpad.optionspace;
 
-public class InputOptionSpace {
+public class OutputOptionSpace {
     public void test() {
-        for (int i = 0; i < 10;) { // violation '';' is not followed by whitespace.'
+        for (int i = 0; i < 10; ) {
         }
         for (int i = 0; i < 10; ) {
         }
