--- src/test/resources/org/checkstyle/autofix/recipe/unusedimports/unusedcasetwo/InputUnusedCaseTwo.java
+++ src/test/resources/org/checkstyle/autofix/recipe/unusedimports/unusedcasetwo/OutputUnusedCaseTwo.java
@@ -9,12 +9,5 @@
 
 package org.checkstyle.autofix.recipe.unusedimports.unusedcasetwo;
 
-import java.io.File; // violation 'Unused import - java.io.File.'
-
-import static java.io.File. // violation 'Unused import - java.io.File.listRoots.'
-    listRoots;
-
-import java.util.List; // violation 'Unused import - java.util.List.'
-
-public class InputUnusedCaseTwo {
+public class OutputUnusedCaseTwo {
 }
