--- src/test/resources/org/checkstyle/autofix/recipe/avoidstarimport/nonstaticfieldandlocalvariablereferences/InputNonStaticFieldAndLocalVariableReferences.java
+++ src/test/resources/org/checkstyle/autofix/recipe/avoidstarimport/nonstaticfieldandlocalvariablereferences/OutputNonStaticFieldAndLocalVariableReferences.java
@@ -9,9 +9,10 @@
 
 package org.checkstyle.autofix.recipe.avoidstarimport.nonstaticfieldandlocalvariablereferences;
 
-import java.util.*; // violation 'Using the .* form of import should be avoided'
+import java.util.ArrayList;
+import java.util.List;
 
-public class InputNonStaticFieldAndLocalVariableReferences {
+public class OutputNonStaticFieldAndLocalVariableReferences {
     private final List<String> values = new ArrayList<>();
     private int count;
 
